device/tests/Microsoft.Azure.Devices.Client.Test/DeviceClientTests.cs

```diff
@@ -27,6 +27,35 @@ public void DeviceClient_OperationTimeoutInMilliseconds_Property_DefaultValue()
         }
 
         [TestMethod]
+        [TestCategory("IoTHubClientDiagnostic")]
+        public void DeviceClient_SetDiagnosticSamplingPercentage()
+        {
+            DeviceClient deviceClient = DeviceClient.CreateFromConnectionString(fakeConnectionString);
+            const int DefaultPercentage = 0;
+            const int ValidPercentage = 80;
+            const int InvalidPercentage = 200;
```

Comment:
Please test both lower and upper limits.

Reply:
fixed